scripts/prepare-release.sh

```diff
@@ -13,8 +13,8 @@ CURR_VERSION=$(node -p "require('./package.json').version")
 IFS='.' read -ra CURR_VERSION_PARTS <<< "$CURR_VERSION"
 
 # Check if a new requiredBefore entry has been added to the data migration manifest
-DIFF_OUTPUT=$(git diff v$CURR_VERSION -- packages/server/src/migrations/data/data-version-manifest.json)
-ADDED_REQUIRED_BEFORE=$(echo "$DIFF_OUTPUT" | grep -e '^\+.*"requiredBefore"')
+DIFF_OUTPUT=$(git diff v$CURR_VERSION -- packages/server/src/migrations/data/data-version-manifest.json) || true
+ADDED_REQUIRED_BEFORE=$(echo "$DIFF_OUTPUT" | grep -e '^\+.*"requiredBefore"' || true)
```

Comment:
Should this fallback with `|| false`?

Reply:
<img width="912" alt="Screenshot 2025-05-02 at 1 52 47 PM" src="https://github.com/user-attachments/assets/3427e2d2-1005-4d1e-99cf-f6e863f90daf" />
Oddly `-z "$(true)"` evaluates to empty